lib/OrganizationsFieldArray/OrganizationCard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  IconButton,
+} from '@folio/stripes/components';
+import OrgInfo from './OrgInfo';
+
+export default class OrganizationCard extends React.Component {
+  static propTypes = {
+    name: PropTypes.string,
+    index: PropTypes.string,
+    organization: PropTypes.object,
+    onDeleteField: PropTypes.func,
+    onReplaceField: PropTypes.func,
+    onAddField: PropTypes.func,
+    organizationPickerComponent: PropTypes.elementType,
```

Comment:
Is it possible to remove this whole `organizationPickerComponent` thing? Would it simplify this whole set of components if we removed it since it looks like we're passing it down several levels? It was only there to get higher coverage stats and it didn't really work out.